internal/common/compilation.bzl

```diff
@@ -301,6 +329,14 @@ def compile_ts(ctx,
           collect_default=True,
           collect_data=True,
       ),
+      "closure_js_library": struct(
```

Comment:
I know you like brevity, but when I'm implementing a spec, I want as much context as physically possible. Since we are implementing the rules_closure closure_js_library interface, I think we should leave the original comments along with a link to the original source.

Reply:
Donezo